src/ExplorerRoutes.tsx

```diff
@@ -29,7 +30,16 @@ export default function ExplorerRoutes() {
           <Route path=":txnHashOrVersion" element={<TransactionPage />} />
           <Route path=":txnHashOrVersion/:tab" element={<TransactionPage />} />
         </Route>
-
+        <Route path="/txn_indexer">
```

Comment:
why is is a separate path? Shouldn't it be able to handle any txn hash or version?